theano/scalar/basic.py

```diff
@@ -604,12 +604,11 @@ def __sub__(self, other):
     def __mul__(self, other):
         return mul(self, other)
 
-    if PY3:
-        def __truediv__(self, other):
-            return div_proxy(self, other)
-    else:
-        def __div__(self, other):
-            return div_proxy(self, other)
```

Comment:
in tensor, \__div__ is always div_proxy. But \__truediv__ is always true_div. There is code that make truediv be div, but it is overrided later.
Without the futur import.
python 2, int / int give and int // int give int.
Python 3, int / int give float and int // int give int.
numpy 1.10 behave like python 2.

With futur import,
python 2 do as python 3
numpy int also do as python 3.

div_proxy always return int_div by default. This is a theano flag operation.

I think we should do as numpy 1.10, we do as python 2/3 depending of the version used and if futur is imported, we behave like python 3. 

It is possible that older version of numpy do differently.

Can you test that with this code in python 2, in a file that just import theano, we keep the python 2 semantic, but in a file where the futur import is there, we use the python 3 semantic?